images/inventory-generator/root/usr/local/bin/user_setup

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -x
+
+# ensure $HOME exists and is accessible by group 0 (we don't know what the runtime UID will be)
+chown ${USER_UID}:0 ${HOME}
+chmod ug+rwx -R ${HOME}
+
+# runtime user will need to be able to self-insert in /etc/passwd
+chmod g+rw /etc/passwd 
+chmod g+rw /opt/app-root/src/.kube/config
+chmod g+rw /opt/app-root/src
+
+# ensure that the ansible content is accessible
+chmod -R g+r ${WORK_DIR} /bin
+find ${WORK_DIR} -type d -exec chmod g+x {} +
+
+# This will redirect us to the latest stable release.
+redirect=$(curl -k https://github.com/openshift/origin/releases/latest)
+# Openshift follows semantic versioning.
+version=$(echo $redirect | grep -o v[0-9]\.[0-9]*\.[0-9]*)
+git_commit=$(git ls-remote https://github.com/openshift/origin refs/tags/$version^{} | awk '{print $1}' | head -c 7)
+
+# Download and extract oc.
+oc_src_dir=openshift-origin-client-tools-$version-$git_commit-linux-64bit
+oc_tar=$oc_src_dir.tar.gz
+wget https://github.com/openshift/origin/releases/download/$version/$oc_tar
+tar -xzf $oc_tar
+rm $oc_tar
+
+mv $oc_src_dir/oc /bin
+rm -r $oc_src_dir
+
+# clone openshift-ansible repository
+git clone https://github.com/openshift/openshift-ansible.git ${HOME}/temp
+cp -r ${HOME}/temp/* ${HOME}/
+rm -rf ${HOME}/temp
+
+# no need for this script to remain in the image after running
```

Comment:
Just hygiene...

Image squash does happen in OSBS. It's not like we're trying to hide what's in the script, just avoiding leaving stuff lying around that isn't useful.